src/admin_access_token.py

```diff
@@ -22,7 +22,7 @@
 SECRET_KEY = "secret-key"  # nosec
 
 
-class AdminAccessTokenService:
+class AdminAccessTokenService:  # pragma: no cover
```

Comment:
Perhaps include a statement here as to why this doesn't need to be covered for unit tests please?

Reply:
Done, I noticed it isn't tested by accident and I annotated it with an additional TODO comment. Thanks!